packages/utils/src/saml/pre-prod-tester.ts

```diff
@@ -0,0 +1,306 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+import { initDbPool } from "@metriport/core/util/sequelize";
```

Comment:
nit: add a `// keep that ^ on top`, so when we use `option + shift + O` to re-order the imports, they don't get lost in the middle